src/Command/Core/CoreConfigureCommand.php

```diff
@@ -104,130 +86,152 @@ protected function execute(InputInterface $input, OutputInterface $output)
+        // Run in a loop to allow multiple selection
+        while (true) {
+            $adapterName = $styleHelper->numericChoice('Choose adapter', $labels);
 
-            if (false === $selection) {
-                $selection = null;
+            if ('noop' === $adapterName) {
+                break;
             }
 
-            $issueTrackerName = $styleHelper->askQuestion(
-                new ChoiceQuestion('Choose issue tracker: ', array_keys($issueTrackers), $selection)
-            );
-        } elseif (null !== $issueTrackerName && !array_key_exists($issueTrackerName, $issueTrackers)) {
-            throw new UserException(
-                sprintf(
-                    'The issue tracker "%s" is invalid. Available adapters are "%s"',
-                    $issueTrackerName,
-                    implode('", "', array_keys($issueTrackers))
-                )
-            );
-        }
+            $this->configureAdapter($input, $output, $adapterName, $adapters[$adapterName]);
+            $this->handleDefaulting($adapterName, $adapters[$adapterName]);
 
-        if (null !== $issueTrackerName) {
-            $this->configureAdapter($input, $output, $issueTrackerName, 'issue_trackers');
-
-            $currentDefault = $this->config->get('issue_tracker');
-            if ($issueTrackerName !== $currentDefault &&
-                $styleHelper->askQuestion(
-                    new ConfirmationQuestion(
-                        sprintf('Would you like to make "%s" the default issue tracker?', $issueTrackerName),
-                        null === $currentDefault
-                    )
-                )
-            ) {
-                $this->config->merge(['issue_tracker' => $issueTrackerName]);
+            if (!$styleHelper->confirm('Do you want configure other adapters?', false)) {
+                break;
             }
         }
 
-        $cacheDir = $styleHelper->askQuestion(
-            (new Question(
-                "Cache folder [{$this->config->get('cache-dir')}]: ",
-                $this->config->get('cache-dir')
-            ))->setValidator(
-                function ($dir) {
-                    if (!is_dir($dir)) {
-                        throw new \InvalidArgumentException('Cache folder does not exist.');
-                    }
-
-                    if (!is_writable($dir)) {
-                        throw new \InvalidArgumentException('Cache folder is not writable.');
-                    }
+        $styleHelper->section('VersionEye configuration');
+        $styleHelper->text(
+            [
+                'VersionEye is a 3rd party service that helps to keep your Composer dependencies up-to-date.',
+                'To use this feature you must first get an API token from: https://www.versioneye.com/settings/api',
+                'The API token is only used for the <info>version-eye:check</info> command.',
+                '',
+                "If you don't want to this service use 'NO_TOKEN' as token."
```

Comment:
You can create a `GushStyle` class that extends from `SymfonyStyle`, then in the `askQuestion` method you just create an instance of `QuestionHelper` instead of `SymfonyQuestionHelper`.
I just think submitting an empty value is quicker and easier than having to type NO_TOKEN